documentation/book/proc-accessing-resource-status.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// master.adoc
+
+[id='proc-accessing-resource-status-{context}']
+= Checking the status of a custom resource
+
+This procedure describes how to find the status of a custom resource.
+
+.Prerequisites
+
+* An {OpenShiftName} cluster
+* A running Cluster Operator
+
+.Procedure
+
+. Specify the custom resource in a standard JSONPath expression to filter on the `status` property:
++
+[source,shell,subs="+quotes,attributes"]
+----
+{cmdcli} get kafka _kafka_resource_name_ -o jsonpath=”{.status}
```

Comment:
I don't think you need the quotes at all because `{status}` won't be interpreted by the shell, however since we're likely to include more jsonpath examples in the future and some of them will need to be quoted to prevent shell interpretation we should probably just use quotes everywhere so users get used to seeing and using them.

```suggestion
{cmdcli} get kafka _kafka_resource_name_ -o jsonpath='{.status}'
```